### PR TITLE
WIP feature branch

### PR DESCRIPTION
**Pull request by JohanLaine**

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 First commit, first row
-First commit, second row
-First commit, third row
-First commit, fourth row
+changed, second row
+changed, third row
```
Comment (JohanLaine) [inline file.txt]: comment here

File: commit_history.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+commit 8be69ee24331eb8ab719e5268241ca57e1f9603f
+Author: Johan Laine (jolae) <jolae@tobii.com>
+Date:   Wed Sep 6 09:46:11 2017 +0200
+
+    WIP feature branch, changed more and rebased
+
+commit 94d69af96d5cf4e9406394cff9dbdb6b04e921e9
+Author: Johan Laine (jolae) <jolae@tobii.com>
+Date:   Wed Sep 6 09:44:51 2017 +0200
```
Comment (JohanLaine) [inline commit_history.txt]: This is true

File: file.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
 First commit, first row
-First commit, second row
-First commit, third row
-First commit, fourth row
+hmm, second row
```
Comment (JohanLaine) [inline file.txt]: More comments on row 2

File: file.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
+hmm, third row
+changed, fourth row
```
Comment (hollste) [inline file.txt]: Fourth row seems weird

File: file.txt (hunk 3 above)
Comment (hollste) [reply file.txt]: good enough

File: commit_history_2.txt [hunk 5]
```diff
@@ -0,0 +1,18 @@
+commit be45b9521e5a3d7ea64723d962cdaaacdb4f3f37
```
Comment (hollste) [inline commit_history_2.txt]: Do we really need two files like these?

File: commit_history_2.txt [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+Author: Johan Laine (jolae) <jolae@tobii.com>
+Date:   Wed Sep 6 09:46:11 2017 +0200
```
Comment (hollste) [inline commit_history_2.txt]: Is this going to the same place as some of the previous comments or new?

File: commit_history.txt [hunk 7]
```diff
@@ -0,0 +1,17 @@
+commit 8be69ee24331eb8ab719e5268241ca57e1f9603f
+Author: Johan Laine (jolae) <jolae@tobii.com>
+Date:   Wed Sep 6 09:46:11 2017 +0200
+
+    WIP feature branch, changed more and rebased
+
```
Comment (hollste) [inline commit_history.txt]: Singleton comment

File: file.txt [hunk 8]
```diff
@@ -1,4 +1,4 @@
-First commit, first row
-First commit, second row
-First commit, third row
-First commit, fourth row
+First commit, 1st row
+hmm, 2nd row
```
Comment (hollste) [inline file.txt]: will be changed

File: file.txt [hunk 9]
```diff
@@ -1,4 +1,4 @@
...
+hmm, 3rd row
```
Comment (hollste) [inline file.txt]: Won't be changed

**Pull request merged**